### PR TITLE
Enable propulsion_types and vehicle_types to be configurable in polic…

### PR DESCRIPTION
**Pull request by janedotx**
…y intents by users

## 📚 Purpose
`propulsion_types` and `vehicle_types` should be configurable by the user.

## 👌 Resolves:
- [ ] 🐛 JRA-834 fixes bug
- [ ] ✨ JRA-756 implements new feature

## 📦 Impacts:
*[list of packages]*

## ✅ PR Checklist
- [ ] Add or remove checklist items to suit your needs

### CODE REVIEW

File: packages/mds-policy-service/tests/policy-client.spec.ts [hunk 1]
```diff
@@ -86,7 +86,9 @@ describe('spot check unit test policy functions with SimplePolicy', () => {
           geographies: [TEST_GEOGRAPHY_UUID1],
           days: ['mon'],
           start_time: '09:00:00',
-          end_time: '10:00:00'
+          end_time: '10:00:00',
+          vehicle_types: ['scooter'],
```
Comment (PeterSP) [inline packages/mds-policy-service/tests/policy-client.spec.ts:90]: Can we also have a test without vehicle_types?

**Review by PeterSP — APPROVED** [review]

File: packages/mds-policy-service/tests/policy-client.spec.ts (hunk 1 above)
Comment (mgeorgehansen-lacuna) [reply packages/mds-policy-service/tests/policy-client.spec.ts:90]: +1

**Review by mgeorgehansen-lacuna — APPROVED** [review]
LGTM! I echo what @PeterSP suggested about adding a test for the edge case where vehicle_types and propulsion types are not provided (or are provided as empty arrays) though.

**Review by nessur — APPROVED** [review]

**Review by PeterSP — APPROVED** [review]

**Review by drtyh2o — APPROVED** [review]
